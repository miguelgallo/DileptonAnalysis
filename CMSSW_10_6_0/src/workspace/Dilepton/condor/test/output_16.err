Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
ImportError: No module named site
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCand
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCandOutOfAccept
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> nGenProtCand
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_status
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_mass
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dpt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dphi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_3Dangle

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00002b5ffdee041c in waitpid () from /lib64/libc.so.6
#1  0x00002b5ffde5df12 in do_system () from /lib64/libc.so.6
#2  0x00002b5ffd7b652d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#3  0x00002b5ffd7b8dd4 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#4  <signal handler called>
#5  0x00002b600a9cee5c in ?? ()
#6  0x3fa47851e0000000 in ?? ()
#7  0xbf5f800000000000 in ?? ()
#8  0xbfb71eb860000000 in ?? ()
#9  0xbf94f5c280000000 in ?? ()
#10 0x3f86c28f60000000 in ?? ()
#11 0xbfb2ae1480000000 in ?? ()
#12 0xbfd1147ae0000000 in ?? ()
#13 0x3fd25999a0000000 in ?? ()
#14 0xbfd03d70a0000000 in ?? ()
#15 0x3fb55c2900000000 in ?? ()
#16 0x3fb98f5c20000000 in ?? ()
#17 0xbf97a8f5c0000000 in ?? ()
#18 0x3fcf4cccc0000000 in ?? ()
#19 0x3fbbfae140000000 in ?? ()
#20 0x3fc53851e0000000 in ?? ()
#21 0x00002b5ffd43d000 in ?? ()
#22 0x0000000000000000 in ?? ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00002b600a9cee5c in ?? ()
#6  0x3fa47851e0000000 in ?? ()
#7  0xbf5f800000000000 in ?? ()
#8  0xbfb71eb860000000 in ?? ()
#9  0xbf94f5c280000000 in ?? ()
#10 0x3f86c28f60000000 in ?? ()
#11 0xbfb2ae1480000000 in ?? ()
#12 0xbfd1147ae0000000 in ?? ()
#13 0x3fd25999a0000000 in ?? ()
#14 0xbfd03d70a0000000 in ?? ()
#15 0x3fb55c2900000000 in ?? ()
#16 0x3fb98f5c20000000 in ?? ()
#17 0xbf97a8f5c0000000 in ?? ()
#18 0x3fcf4cccc0000000 in ?? ()
#19 0x3fbbfae140000000 in ?? ()
#20 0x3fc53851e0000000 in ?? ()
#21 0x00002b5ffd43d000 in ?? ()
#22 0x0000000000000000 in ?? ()
===========================================================


