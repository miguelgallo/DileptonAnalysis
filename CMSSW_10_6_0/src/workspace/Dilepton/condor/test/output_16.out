copying file
running
0cortes: 1454886
1cortes: 20616
2cortes: 19121
3cortes: 18308
4cortes: 8489
5cortes: 8489
6cortes: 4989
7cortes: 726
8cortes: 10
moving file
